adapters/dmx/dmx.go

```diff
@@ -271,10 +278,15 @@ func fetchParams(params dmxExt, inst openrtb.Imp, ins openrtb.Imp, imps []openrt
 		tempimp.Secure = &intVal
 	}
 	if banner != nil {
+		if banner.H == checkValue || banner.W == checkValue {
```

Comment:
I'm unclear what you're looking to accomplish with this check. The `checkValue` is never initialized, so isn't this the same as `if banner.W == nil || banner.W == nil {`?